ui/src/app/components/webcomponents/left-nav.component.html

```diff
@@ -4,68 +4,91 @@
     <a [routerLink]="['']">
     </a>
   </div>
-  <div class="text-center dropright mouse-over" (mouseover)="displayGlobalAdd=true" (mouseleave)="displayGlobalAdd=false">
-    <a class="primary-icon primary-icon-padding" href="javascript:void(0)">
+  <div class="text-center">
+    <div class="p-12"></div>
+    <div class="dropright mouse-over" (mouseover)="displayGlobalAdd=true" (mouseleave)="displayGlobalAdd=false">
+      <a class="primary-icon p-0" href="javascript:void(0)" style="height: unset">
       <div>
         <i class="fa-plus-filled"></i>
       </div>
     </a>
     <app-global-add
       *ngIf="authGuard?.session?.user && displayGlobalAdd"
-      class="global-add-dropdown"></app-global-add>
+      (isClicked)="displayGlobalAdd = $event"
+      class="global-add-dropdown mt-n10"></app-global-add>
+    </div>
+    <div class="p-6"></div>
   </div>
-  <ul class="primary-nav mt-0" id="ul_primary_menu">
-    <li class="pt15 separator mb-30">
-    </li>
-    <li [routerLink]="['/dashboard']" [routerLinkActive]="'active'">
+  <div>
+    <div class="pt15 mb-30" style="
+        border-bottom: solid 1px #01593D !important;
```

Comment:
plz avoid inline styles and try to not use important unless must required

Reply:
made changes